### PR TITLE
Add Empty state when workItems are not available

### PR DESCRIPTION
**Pull request by sahil143**
Changes:

- Hide 1st column when user is not Logged In
- Add Empty state
- Fix Detail view route Url

### CODE REVIEW

**Comment by fabric8cd** [comment]
@sahil143 fabric8/fabric8-ui:SNAPSHOT-PR-2398-1 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2398-fabric8-planner.badger.fabric8.io

File: src/app/components/planner-list/planner-list.component.html [hunk 1]
```diff
@@ -300,6 +301,7 @@ <h4>Please wait, we are loading your data.</h4>
             </ng-template>
           </ngx-datatable-column>
         </ngx-datatable>
+        <pfng-empty-state *ngIf="datatableWorkitems?.length <= 0 && !uiLockedList" [config]="emptyStateConfig"></pfng-empty-state>
```
Comment (sanbornsen) [inline src/app/components/planner-list/planner-list.component.html]: Break them into two lines

File: src/app/components/planner-list/planner-list.component.ts [hunk 2]
```diff
@@ -1057,7 +1066,7 @@ export class PlannerListComponent implements OnInit, AfterViewChecked, OnDestroy
   onDetailPreview(id): void {
     event.stopPropagation();
     this.workItemDataService.getItem(id).subscribe(workItem => {
-       this.router.navigateByUrl(this.router.url.split('/list')[0] + '/detail/' + workItem.id, { relativeTo: this.route });
+       this.router.navigateByUrl(this.router.url.split('plan')[0] + 'plan/detail/' + workItem.id, { relativeTo: this.route });
```
Comment (sanbornsen) [inline src/app/components/planner-list/planner-list.component.ts]: Break it into lines

**Comment by fabric8cd** [comment]
@sahil143 fabric8/fabric8-ui:SNAPSHOT-PR-2398-4 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2398-fabric8-planner.badger.fabric8.io

**Comment by rgarg1** [comment]
hidden add workitem option

**Comment by rgarg1** [comment]
@sahil143 Already approved. The scope of my finding is much larger and will be addressed outside this PR.

**Comment by fabric8cd** [comment]
@sahil143 fabric8/fabric8-ui:SNAPSHOT-PR-2398-6 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2398-fabric8-planner.badger.fabric8.io

**Comment by rgarg1** [comment]
LGTM @sahil143. 

The inline quick-add button is now displayed, but disabled.

**Pull request merged**